doc/changes/latest.inc

```diff
@@ -148,6 +148,8 @@ Bug
 
 - Fix the ``reduce_rank`` parameter in :func:`mne.beamformer.make_lcmv` and :func:`mne.beamformer.make_dics` to reduce the rank of the leadfield first before then reducing the rank of the denominator of the beamformer formula for the inversion by `Britta Westner`_ and `Eric Larson`_.
 
+- Fix bug with :func:`mne.beamformer.make_lcmv` with MaxFiltered data where rank was incorrectly checked by `Eric Larson`_.
+
```

Comment:
I think it is not only for maxfiltered data. Could also happen with ICA'ed data etc - if no noise covariance matrix is supplied.